src/components/v1/V1Project/BalanceTimeline.tsx

```diff
@@ -93,6 +93,7 @@ export default function BalanceTimeline({ height }: { height: number }) {
         duration,
         false,
       )
+      // @ts-ignore
```

Comment:
@peripheralist -- Not sure how this was working. Assuming there's some kind of mismatch in the types or something?


I downloaded `"@types/ethereum-block-by-date": "^1.4.1",` types, but now that I think of it I'm really curious if there was a reason why they weren't initially added.


Just wanted to call this to your attn.